Source/player.h

```diff
@@ -372,6 +372,8 @@ struct Player {
 
 	bool CanUseItem(const Item &item) const
 	{
+		if (item.IDidx == IDI_SPECELIX && Quests[Q_MUSHROOM]._qactive != QUEST_DONE)

```

Comment:
i guess the down side is that if you bring the elexier with you in to a new game it can no longer be used. Still that is better then the current situation.

Reply:
The worst part of the old behavior has many disadvantages even in singleplayer when you finished the quest:
- when you didn't use the elixir, saved and restarted the application you couldn't use it
- when you loaded a old save state where you had the elixir but didn't finished the quest you could use it and got a broken quest